design/scss/legacy/utils.scss

```diff
@@ -114,16 +114,16 @@ Updated buttons
 	-o-border-radius: 0.5rem;
 	font-size: 1.25rem;
 	border: none;
-	font-weight: $semi-bold;
+	font-weight: $fw-semi-bold;
 }
 
 .button-accent {
 	@extend %btn;
 	@extend %btn-updated;
-	background-color: $custom-orange;
+	background-color: $secondary;
```

Comment:
Can you change the variable name from $secondary to $accent? 
I'm trying to have 3 buttons (button-accent for any orange bg, button-primary for navy bg, button-secondary for unimportant things like "log out" button on profile page, which is gray bg with navy color)